ajax/libs/typeboost.css/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "typeboost.css",
+  "filename": "typeboost.min.css",
+  "version": "0.1.8",
+  "description": "typeboost.css as a node packaged module",
```

Comment:
Sorry that I can't understand this description. I think you can make a better one.

But since this is same as origin, I've no reason to reject.